File: src/main/java/com/rkroom/blog/service/SiteService.java
package com.rkroom.blog.service;

import com.rkroom.blog.entity.Site;

import java.util.List;
import java.util.Map;

public interface SiteService {
    List<Site> selectAll();
    Map selectOpenInfo();
    void changeValueById(String value,int id);
    void insertSite(Site site);
    Site selectByAttribute(String attribute);
}

File: src/main/java/com/rkroom/blog/service/SiteService.kt
package com.rkroom.blog.service

import com.rkroom.blog.entity.Site

interface SiteService {
    fun selectAll(): List<Site?>?
    fun selectOpenInfo(): Map<*, *>?
    fun changeValueById(value: String?, id: Int)
    fun insertSite(site: Site)
    fun selectByAttribute(attribute: String?): Site?
}